remoulade/state/backends/postgres.py

```diff
@@ -125,28 +153,66 @@ def get_states(
         end_datetime: Optional[datetime.datetime] = None,
         sort_column: Optional[str] = None,
         sort_direction: Optional[str] = None,
-        get_groups: bool = False,
     ):
+        sort_column = sort_column or "enqueued_datetime"
```

Comment:
you can put them in the default instead of None directly no? plus it will avoid the optional type

Reply:
The problem is that when a user passed None in the api, the api will pass None to this function and the default value will not be used. There are 2 main ways of enforcing the default value, the first being what I am doing, and the second would be using something like post_dump in the schema (as the missing option would not replace a specified None value), which seemed more complicated to me, which is why I chose to do this.